domains/jordan.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "eaglenetwork",
+  },
+  "records": {
+    "A": "146.190.116.231"
```

A records is an array
```suggestion
    "A": ["146.190.116.231"]
```